src/components/IndexTable/components/Row/Row.tsx

```diff
@@ -32,22 +32,17 @@ export const Row = memo(function Row({
   status,
   onNavigation,
 }: RowProps) {
-  const {selectMode, condensed} = useIndexRow();
+  const {selectable, selectMode, condensed} = useIndexRow();
   const onSelectionChange = useIndexSelectionChange();
   const {
     value: hovered,
     setTrue: setHoverIn,
     setFalse: setHoverOut,
   } = useToggle(false);
 
-  const rowClassName = classNames(
-    styles.TableRow,
-    condensed && styles.condensedRow,
-    selected && styles['TableRow-selected'],
-    subdued && styles['TableRow-subdued'],
-    hovered && styles['TableRow-hovered'],
-    status && styles[variationName('status', status)],
-  );
+  const getPrimaryLinkElement = () => {
```

Comment:
[nit] this should maybe be memoized since (1) it's doing a DOM lookup, (2) it's called 3x on each rerender, and (3) the value isn't that likely to change on rerender

Reply:
Good idea– it's a little tricky to memoize the function here since its dependencies aren't based on a state value, but I updated it to use a callback ref which will set `primaryLinkElement.current` only once when the `tableRowRef` mounts, which should have the same effect.